snap/snapcraft.yaml

```diff
@@ -29,20 +40,20 @@ parts:
   ddgr:
     plugin: make
     source: https://github.com/jarun/ddgr.git
-    override-pull: |
-      snapcraftctl pull
-      last_committed_tag="$(git describe --tags --abbrev=0)"
-      last_released_tag="$(snap info ddgr | awk '$1 == "latest/beta:" { print $2 }')"
-      # If the latest tag from the upstream project has not been released to
-      # beta, build that tag instead of master.
-      if [ "${last_committed_tag}" != "${last_released_tag}" ]; then
-        git fetch
-        git checkout "${last_committed_tag}"
-      fi
+# Commenting this out - should be removed if it doesn't break anything.
```

Comment:
Can we confirm if this is actually redundant and just remove it? Looks like it's safe as it was being used to derive a version - which we no longer need here 😊

Reply:
Saying that, do we not need some sort of `source-tag` or similar to make sure the right version is being cloned before the build?